InFact-coop/YiMovi
# Create csv 2

Script that takes all json files in `/locales` and transforms them to a `.csv` formatted file which we can send to a translator.
Related #137 

To test this:
- make sure you have files for en and chn 
- run `node lib/parse_json_csv.js`
